src/main/kotlin/no/nav/familie/tilbake/api/dto/ManuellBrevmottakerDto.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.familie.tilbake.api.dto
+
+import no.nav.familie.kontrakter.felles.tilbakekreving.Vergetype
+import no.nav.familie.tilbake.dokumentbestilling.manuell.brevmottaker.domene.MottakerType
+import java.util.UUID
+
+data class ManuellBrevmottakerDto(
+    val id: UUID? = null,
+    val type: MottakerType,
+    val navn: String,
+    val adresselinje1: String,
+    val adresselinje2: String?,
+    val postnummer: String,
+    val poststed: String,
+    val landkode: String,
+    var vergetype: Vergetype? = null
```

Comment:
Har ikke satt meg inn i Brevmottaker verden, men er det noen grunner for at vergetype må være var i en dto?